app/indexers/default_object_rights_indexer.rb

```diff
@@ -13,7 +13,9 @@ def to_solr
 
     {
       'use_statement_ssim' => use_statement,
-      'copyright_ssim' => copyright
+      'copyright_ssim' => copyright,
+      'rights_descriptions_ssim' => 'Dark',
+      'default_rights_descriptions_ssim' => RightsDescriptionBuilder.build(cocina)
     }
```

Comment:
APOs are always "Dark" for their own rights, and get a new field for user display of default object rights